### ISSUE TITLE
Issues with ptcgoCodes

### ISSUE THREAD

**Issue by MatiasOETamminen** [PokemonTCG/pokemon-tcg-data#184]
Firstly, I think the ptcgoCode is a slight misnomer, as only HS-on exist in PTCGO. I guess we're trying to reach some kind of "official" abbreviations for all the sets. There are a couple of hurdles to overcome with this:

1. The official tournament rules (https://www.pokemon.com/us/play-pokemon/about/tournaments-rules-and-resources/) and the abbreviations therein differ slightly from the abbreviations PTCGO uses, mainly with promos:

Take smp-sm107 for example – this is how PTCGO exports it:
* 1 Dusk Mane Necrozma PR-SM 107
so set = PR-SM
number = 107

This is what the physical TCG uses: 
Dusk Mane Necrozma PR-SM107
so set = PR
number = SM107

We currently have this strange frankenstein:
set = PR-SM
number = SM107

So basically PTCGO splits the set from the number at a different point. Also, physical TCG uses PR-BW### whereas PTCGO uses PR-BLW ###.

2. There are no undisputed sources on the official abbreviations pre-BLW (or pre-HS if you count PTCGO). What's the source on the current abbreviations of these old sets?

3. Some of the existing sets are missing this parameter (I can add these if the abbreviations are acceptable):

- Southern Islands (should probably be SI)
- all the POP sets (should probably be POP#)
- Pokémon Rumble (RM?)
- all the McDonald's Collections (each of these are officially MCD, does the ptcgoCode value need to be unique? Also, ALL promos at least from BLW onwards are just PR in the physical TCG rules).

**Comment by immewnity** [comment]
Just wanted to chime in here - I'm going to get a group together across several sites/services to try and standardize the codes bring used, to help with interoperability. Should be posting about it in the Discord within the week.

**Comment by MatiasOETamminen** [comment]
> Just wanted to chime in here - I'm going to get a group together across several sites/services to try and standardize the codes bring used, to help with interoperability. Should be posting about it in the Discord within the week.

Cool! Just hoping it won't end up like this: https://xkcd.com/927/ :D

**Comment by immewnity** [comment]
Same here 😅

**Comment by adback03** [comment]
@immewnity did you ever post anything about this?
